### DISCUSSION TITLE
How to clean up failed flexget installation

### DISCUSSION THREAD

**Discussion by Anthuan** [alexbelgium/hassio-addons#461] [Q&A] [answered]
How do I clean up my homeassistant instance to be able to re-install? This is what I did
1) installed flexget (no errors)
2) tried to log in to the web interface, asked me to set the password
3) googled it, looked like I needed to do a flexget command from the CLI
4) couldn't figure out how to log on to the host where flexget was running, so I decided to re-install to see if maybe I skipped a step during installation
5) uninstalled ok, BUT
6) when I try to install now, I get this error message:

Failed to install add-on
The command '/bin/bash -o pipefail -c grep -sril "Potential tampering with custom" /etc/cont-init.d /etc/s6-overlay/s6-rc.d | xargs -r rm && grep -sril "/config" /etc/cont-init.d /etc/s6-overlay/s6-rc.d | xargs grep -riL "/config/" | xargs -r rm && for file in $(grep -srl "PUID:-911" /etc/cont-init.d /etc/s6-overlay/s6-rc.d); do sed -i 's/bash/bashio/g' "$file" && sed -i 's/{PUID:-911}/(bashio::config "PUID")/g' "$file" && sed -i 's/{PGID:-911}/(bashio::config "PGID")/g' "$file"; done && grep -rl " /config" /etc/cont-init.d | xargs sed -i 's| /config| /config/addons_config/flexget|g' && grep -rl " /config" /etc/services.d | xargs sed -i 's| /config| /config/addons_config/flexget|g' && sed -i 's=/config/flexget=/config/addons_config/flexget || true=g' /etc/cont-init.d/10-adduser && sed -i 's/bash/bashio/g' /etc/cont-init.d/20-config && sed -i 's/# set FG_WEBUI_PASSWD/if bashio::config.has_value "WebuiPass"; then FG_WEBUI_PASSWD=$(bashio::config "WebuiPass"); else FG_WEBUI_PASSWD=""; fi/g' /etc/cont-init.d/20-config && sed -i 's/# install custom plugins/if bashio::config.has_value "FG_PLUGINS"; then FG_PLUGINS=$(bashio::config "FG_PLUGINS"); else FG_PLUGINS=""; fi/g' /etc/cont-init.d/20-config' returned a non-zero code: 1

I'd like to undo whatever it is that I did, so I can try again. I'm running homeassistant from a containers (I was expecting less headache for restarts and updates), if that makes any difference. Any pointers on where I should start?

**Comment by alexbelgium** [comment, marked as answer]
Hi thanks for your report. I introduce automatic addons update based on upstream change. What that means is that when the owner of flexget issues a new version, my addon is automatically updated. However, it can cause issues if he has modified the structure of the container he provides. This is the case. I've updated the addon and it should install fine

The webui password can be set with the addon option WebuiPass

If you need a full uninstall, you should delete /config/addons_config/flexget

**Comment by Anthuan** [comment]
Looks like that did it! I installed it successfully. I'm still having trouble starting it. Looking at the logs, looks like I have a file missing:

cont-init: info: /etc/cont-init.d/10-adduser exited 0
cont-init: info: running /etc/cont-init.d/15-install-pkg
cont-init: info: /etc/cont-init.d/15-install-pkg exited 0
cont-init: info: running /etc/cont-init.d/20-config
/config/addons_config/flexget
*** Using existing config.yml
*** Setting flexget web password
Could not instantiate manager: Config `/config/addons_config/flexget` does not appear to be a file.
cont-init: info: /etc/cont-init.d/20-config exited 1
cont-init: info: running /etc/cont-init.d/90-custom-folders
cont-init: info: /etc/cont-init.d/90-custom-folders exited 0
cont-init: info: running /etc/cont-init.d/99-custom-files
cont-init: info: /etc/cont-init.d/99-custom-files exited 0
cont-init: warning: some scripts exited nonzero
s6-rc: warning: unable to start service legacy-cont-init: command exited 1
/run/s6/basedir/scripts/rc.init: warning: s6-rc failed to properly bring all the services up! Check your logs (in /run/uncaught-logs/current if you have in-container logging) for more information.
/run/s6/basedir/scripts/rc.init: fatal: stopping the container.
s6-rc: info: service fix-attrs: stopping
s6-rc: info: service 00-legacy: stopping
s6-rc: info: service fix-attrs successfully stopped

I will keep googling on how to recreate that missing file, and see what else I can do. Thank you for the help!

**Reply by alexbelgium** [reply]
Hi, it should be good now, I've pushed a new version !
